Resources/Prototypes/Entities/Objects/Devices/translators.yml

```
@@ -219,3 +219,20 @@
     requires:
     - TauCetiBasic
     setLanguageOnInteract: false
+
+- type: entity
+  id: ValyrianStandardTranslator
+  parent: [ TranslatorPoweredBase ]
```

yamllint

[error] too many spaces inside brackets

(brackets)